apps/files/js/file-upload.js

```diff
@@ -213,6 +213,15 @@ OC.FileUpload.prototype = {
 		var data = this.data;
 		var file = this.getFile();
 
+		/**
+		 * isDirectory indicates an empty directory, so we just check if it
+		 * already exists and create one if not.
+		 * After we exit the function, since no data needs to be uploaded.
```

Comment:
```suggestion
		 * Afterward, we exit the function, since no data needs to be uploaded.
```

This seems a bit clearer to me, rather than "After we exit the function, ..." which means something different.